src/js/letters/containers/AddressSection.jsx

```diff
@@ -82,6 +85,14 @@ export class AddressSection extends React.Component {
           {invalidAddressProperty}
         </div>
       );
+    // If countries and states are not available when they try to update their address,
+    // they will see this warning message instead of the address fields.
```

Comment:
This only happens if
1. There's a problem between the api and EVSS and the api no longer has these values in the cache
2. We haven't gotten the response back from the api yet
    - So...how does that play with slow connections? Will this warning message pop in while we're waiting?

Reply:
The fetch is attempted in `Main`, so I assume we'll get a response back from that fetch before the user is able to click the `Update` button and `isEditingAddress` is changed to `true`, especially because the address content won't render until the fetch for the address is also returned (which happens at the same time as the fetch for the countries and states). I wonder if we should tie the loading indicator for the whole app on the responses from *all* the fetches in `Main`. What do you think @cvalarida and @markgreenburg?

Reply:
Created a new ticket to handle this separately: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4990